src/_includes/layouts/default.njk

```diff
@@ -27,6 +27,7 @@
           <li><a href="/code-of-conduct">Code of Conduct</a></li>
           <li><a href="/financial-reports">Financial Reports</a></li>
           <li><a href="/meeting-notes">Meeting Notes</a></li>
+          <li><a href="/project-guideline">Project Guideline</a></li>
```

Comment:
I believe this should be plural, I.e. `project-guidelines`.

I also think that this isn't important enough in the [information hierarchy](https://www.bridgewaterlearning.co.za/2013/04/16/design-principles-hierarchy-of-information/) to be it's own top-level page, and maybe it belongs within the `/maintaining` page that I am sprouting in #28 ?

Reply:
I agree it should be plural